.gitignore

```diff
@@ -18,3 +18,5 @@ obp-api/src/main/resources/log4j.properties
 obp-api/src/main/scripts/kafka/kafka_*
 obp-api/src/main/scripts/kafka/logs/
 obp-api/src/main/scripts/kafka/tmp/
+/obp-commons/src/main/resources/git.properties
+/obp-api2/
```

Comment:
the /opb-api2 is a local project to find which scala file cause memory error. It always cause wrong commit, so here add to .gitignore to avoid wrong commit.